xbrowse_server/matchmaker/utilities.py

```diff
@@ -29,18 +32,21 @@ def get_all_clinical_data_for_family(project_id,family_id,indiv_id):
             A JSON object as per MME spec of a patient
     """
     project = get_object_or_404(Project, project_id=project_id)
+    
+    #get contact information if available
+    seqr_project = project.seqr_project 
+    #Ideally these (the default Sam, and matchmaker@broad) should be set at project creation   
+    seqr_project.is_mme_enabled=True 
+    seqr_project.mme_primary_data_owner=settings.MME_CONTACT_NAME
+    seqr_project.mme_contact_url=settings.MME_CONTACT_HREF
 
     #species (only human for now) till seqr starts tracking species
     species="NCBITaxon:9606"
-
-    #contact (this should be set in settings
-    href=settings.MME_CONTACT_HREF
-    if settings.MME_PATIENT_PRIMARY_DATA_OWNER[project_id]["email"] != "":
-        href = href + ',' + settings.MME_PATIENT_PRIMARY_DATA_OWNER[project_id]["email"]
+    
     contact={
```

Comment:
yup, they are,
```
    #add new patient to affected patients
    affected_patient={"id":id,
                      "species":species,
                      "label":label,
                      "contact":contact,
                      "features":features,
                      "sex":sex,
                      "genomicFeatures":genomic_features
                      }
```
this maps straight to the MME contact field of patient

Reply:
hmm..the logic needs this, but if you hate doing this here, we could put it in the model as a constant too at seqr.models.Project

class Project(ModelWithGUID):
....
    #primary_investigator = models.ForeignKey(User, null=True, blank=True, related_name='+')

    is_phenotips_enabled = models.BooleanField(default=False)
    phenotips_user_id = models.CharField(max_length=100, null=True, blank=True, db_index=True)

    is_mme_enabled = models.BooleanField(default=False)
    mme_primary_data_owner = models.TextField(null=True, blank=True)
    mme_contact_url = models.TextField(null=True, blank=True)

    seqr_project.is_mme_enabled=True 
    seqr_project.mme_primary_data_owner=settings.MME_CONTACT_NAME
    seqr_project.mme_contact_url=settings.MME_CONTACT_HREF

....

this might be more elegant..

Reply:
Sorry I'm still confused. The way the logic is now with .save(), every time you call the get_submission_candidates API, it will reset mme_primary_data_owner and mme_contact_url to the defaults. Wouldn't this overwrite any changes the user made to these?

Reply:
Let's say the user changes these for one submissions, I think the new values should persist on subsequent submissions.